README.asciidoc

```diff
@@ -316,15 +316,12 @@ The syntax for such an entry is
 
 [source,toml]
 ----
-[[semantic_tokens]]
-token = "variable"
-face = "const_variable_declaration"
-modifiers = ["constant", "declaration"]
+"[token][+modifier1[+modifier2]...]" = "face"
```

Comment:
Looks reasonable. This is a nice improvement, and validation is still fairly easy.

Regarding the breakage, we probably should have left these commented in `kak-lsp.toml`, or maybe extend dynamic configuration options, so that most (casual) users don't need a `kak-lsp.toml`.

The only thing I'm not sure about is that given `token+modifier1+modifier2 = face`, the `token` does not really stand out.
I considered both

- `token+modifier1,modifier2 = face`
- `token+modifier1|modifier2 = face`

but since precedence is not obvious, your version may be better, not sure.
The `+` is a good choice because it's also used for Kakoune attributes.

Reply:
I get the syntax concern, but I think the fact that it matches kakoune attributes syntax, combined with the simplicity, makes it a good option.

Regarding dynamic configuration, I haven't used it personally, and don't know a lot about it. But it might make sense to do something special for this as well